### ISSUE TITLE
Visible seam caused by repeating texture

### ISSUE THREAD

**Issue by AldeRoberge** [BLaZeKiLL/VloxyEngine#105]
![image](https://user-images.githubusercontent.com/4276275/236240754-e6c3b3bd-e665-419c-a1d9-71c9abf05eec.png)

Hello, upon closer inspection there seems to be a visible seam.

**Comment by AldeRoberge** [comment]
Possible fix :

![image](https://user-images.githubusercontent.com/4276275/236244895-bf42972d-8cc3-4c92-9f89-9ccf3d6ebb82.png)

**Comment by AldeRoberge** [comment]
Possible fix : Change the Sampler in TextureArray_AO to use Mirror instead of Repeat 
![image](https://user-images.githubusercontent.com/4276275/236245364-1e2111aa-eb93-4063-97ea-edde400833e0.png)

**Comment by BLaZeKiLL** [comment]
Yeah this had been annoying me, thanks you can create a pull request for this if you like
